dsi-iepg/cas-connection/0.1/manifest.json

```
@@ -0,0 +1,30 @@
+{
+    "bundles": {
+        "Iepg\\Bundle\\Cas\\CasConnectionBundle": ["all"]
+    },
+    "copy-from-recipe": {
+        "config/": "%CONFIG_DIR%/",
+        "src/": "%SRC_DIR%/"
+    },
+    "env": {
+        "#1": "#### Parameters for CAS connection ###",
+        "CAS_HOST": "cas-adresse.com",
+        "#2": "#This value is optional",
```

```suggestion
        "#2": "# This value is optional",
```

Same for all commented lines.